### DISCUSSION TITLE
Hepl Installer token has expired. Please generate a new one RMM

### DISCUSSION THREAD

**Discussion by lpoiie** [amidaware/tacticalrmm#1098] [Q&A] [answered]
Hello
I need help please
When I want install Tactical  RMM client on computer that message appear : Installer token has expired. Please generate a new one

How can resolve  this problem ?
Tanks for your help.
![278795992_2689139291219112_8632880254257473791_n](https://user-images.githubusercontent.com/104449614/165357074-2d994a17-4571-4145-a51d-aab454f71b88.jpg)

**Comment by BelleNottelling** [comment, marked as answer]
When you generwte an agent, it specifically says that the token will be valid for 24 hours.
At that point you need a new installer

**Comment by lpoiie** [comment]
thank you for your reply
I'm not sure I understand. It's been like this for a few days, since I renewed the security certificate. Can you tell me how to proceed please?

**Reply by dinger1986** [reply]
Read the docs

https://docs.tacticalrmm.com/install_agent/

Deployment links is what you are using 

For security you do not want them to be set for too long!

**Comment by lpoiie** [comment]
Tanks
I Genered another agent and the problem resovled
:-)
